attestation-agent/deps/crypto/src/rand.rs

```diff
@@ -0,0 +1,7 @@
+use rand::Rng;
+
+pub fn random_key<const N: usize>() -> Vec<u8> {
```

Comment:
AFAUI this intents to be a generic helper but the name sounds specific to the use LUKS2 case

```suggestion
pub fn random_bytes<const N: usize>() -> Vec<u8> {
```

(or make it local?)